components/pipeline-service/development/bump-results-watcher-replicas.yaml

```diff
@@ -1,4 +1,4 @@
 - op: replace
   path: /spec/replicas
   # default pipeline-service setting is 1
-  value: 2
\ No newline at end of file
+  value: 1
```

Comment:
been backing off of multiple replicas since some odd behavior I saw during the March outage.

production is already back to 1 replica